CMakeLists.txt

```diff
@@ -138,7 +173,19 @@ if (MINGW)
         -fdata-sections;          # One data per section (enables GC)
       >
     )
-    target_link_options(OpenSCAD PRIVATE
+    target_link_options(OpenSCADLibInternal PRIVATE
+      $<$<CONFIG:Debug>:
+        -Wl,--gc-sections;        # Linker: drop unused sections
+      >
+    )
+    # Unsure if it helps to double this
+    target_link_options(OpenSCADExe PRIVATE
+      $<$<CONFIG:Debug>:
+        -Wl,--gc-sections;        # Linker: drop unused sections
+      >
+    )
+    # Unsure if it helps to triple this
```

Comment:
The static linker likely ignores those options, but a future dynamic linker may not. I'm unsure as well.

Reply:
I'll leave it for the brave soul who wishes to make a shared library.